### DISCUSSION TITLE
Instructions for installation via brew

### DISCUSSION THREAD

**Discussion by wAuner** [jotaen/klog#255] [Help / Question]
Hi,

I was wondering why there is no mention of installing klog via brew in the macOS installation guide?
I would expect this to be the primary and most logical way for most terminal users. I’ve been using klog for 2 years and just found out that there is a brew tap. Is it working?
And wouldn‘t it be possible to add klog to brew core instead of a private tap? AFAICT klog should be a good candidate.

**Comment by jotaen** [comment]
Yes, the [brew tap](https://github.com/jotaen/homebrew-klog) (`brew tap jotaen/klog`) should work on both Intel and ARM Macs.

I realized that the structure around the install instructions were a bit messy, which [I fixed here](https://github.com/jotaen/klog/pull/256).

It should theoretically be possible to submit klog to the brew core, but I’m reluctant to do that for a number of reasons. In regards to Homebrew specifically, I don’t use it, and also know very little about it, so I rely on other people to help me maintain the tap (or even get it to run in the first place). My other issue is that creating klog install packages and submitting them to Homebrew (but then also apt, yum, dnf, pacman, Chocolatey, MacPorts, scoop, etc. pp.) requires a lot of extra time and work. I’m aware that the manual install process is not super convenient for end-users, but I still decided to draw a line there in regards to what I’m able to provide. The Homebrew tap is a bit of an exception here, but that mostly for “historic” reasons.
